src/components/navBar.js

```diff
@@ -1,23 +1,27 @@
-import Logo from "../assets/spacecenterlogo.svg";
+import Logo from "../assets/logo.png";
 import { Link } from "react-router-dom";
 
 function NavBar() {
   return (
-    <div className="bg-mine-shaft items-center p-4 flex gap-8 max-h-28">
+    <div className="bg-navbar-gray items-center p-4 flex gap-8 max-h-16">
       <Link to="/home">
-        <img
-          className=" p-8 items-center gap-8"
-          src={Logo}
-          alt="Space Center Logo"
-        />
+        <img className="ml-4 mr-60" src={Logo} alt="Space Center Logo" />
       </Link>
+
       <Link to="/home">
-        <h1 className="text-white text-3xl">
-          {" "}
-          Space
-          <br />
-          Center
-        </h1>
+        <button className="text-navbar-components-gray ml-96 mr-8">Home</button>
+      </Link>
+
+      <Link to="/question">
+        <button className="text-navbar-components-gray mr-8">
```

Comment:
You don't need a button inside a Link. The Link component renders an `<a>` tag, which is for navigation, so it's what we want here. `<button>` is used for other interactive things, but you don't need both at the same time.